cg_hermes/config/balsamic.py

```diff
@@ -1,627 +1,487 @@
-    },
-    frozenset({"vcf-all", "tnhaplotyper", "snv", "annotated-somatic-vcf-all-index"}): {
-        "tags": ["vcf-index", "tumor", "haplotype-caller", "somatic"],
-        "is_mandatory": True,
-        "used_by": ["storage"],
-    },
-    frozenset({"vcf-filtered", "snv", "research-vcf-filtered"}): {
-        "tags": ["vcf-snv-research", "filtered", "somatic", "haplotype-caller"],
-        "is_mandatory": False,
-        "used_by": ["storage"],
-    },
-    frozenset({"vcf-filtered", "snv", "research-vcf-filtered-index"}): {
-        "tags": ["vcf-snv-research-index", "filtered", "somatic", "haplotype-caller"],
-        "is_mandatory": False,
-        "used_by": ["storage"],
+        "used_by": ["audit"],
     },
-    frozenset({"snv", "research-vcf-pass", "vcf-pass"}): {
-        "tags": ["vcf", "filtered", "somatic", "haplotype-caller"],
+    # Alignment files
+    frozenset(TAGS["tumor.merged.cram"]): {  # cram (tumor)
+        "tags": ["tumor", "cram"],
         "is_mandatory": True,
-        "used_by": ["storage"],
+        "used_by": ["deliver", "scout"],
     },
-    frozenset({"snv", "research-vcf-pass-index", "vcf-pass"}): {
-        "tags": ["vcf-index", "filtered", "somatic", "haplotype-caller"],
+    frozenset(TAGS["tumor.merged.cram.crai"]): {
+        "tags": ["tumor", "cram-index"],
         "is_mandatory": True,
-        "used_by": ["storage"],
+        "used_by": ["deliver", "scout"],
     },
-    frozenset({"vcf-all", "tnscope", "snv", "annotated-somatic-vcf-all"}): {
-        "tags": ["tumor", "tnscope", "vcf-snv-research", "somatic"],
+    frozenset(TAGS["normal.merged.cram"]): {  # cram (normal)
+        "tags": ["normal", "cram"],
         "is_mandatory": False,
-        "used_by": ["scout", "deliver"],
+        "used_by": ["deliver", "scout"],
     },
-    frozenset({"vcf-all", "tnscope", "snv", "annotated-somatic-vcf-all-index"}): {
-        "tags": ["tumor", "tnscope", "vcf-snv-research-index", "somatic"],
+    frozenset(TAGS["normal.merged.cram.crai"]): {
+        "tags": ["normal", "cram-index"],
         "is_mandatory": False,
-        "used_by": ["scout", "deliver"],
-    },
-    frozenset({"vcf-all", "manta", "annotated-somatic-vcf-all", "sv"}): {
-        "tags": ["vcf-sv-research", "manta", "tumor", "somatic"],
-        "is_mandatory": True,
-        "used_by": ["scout", "deliver"],
+        "used_by": ["deliver", "scout"],
     },
-    frozenset({"vcf-all", "annotated-somatic-vcf-all-index", "manta", "sv"}): {
-        "tags": ["vcf-sv-research-index", "manta", "tumor", "somatic"],
-        "is_mandatory": True,
-        "used_by": ["scout", "deliver"],
-    },
-    frozenset({"vcf-all", "cnvkit", "cnv", "annotated-somatic-vcf-all"}): {
-        "tags": ["cnvkit", "sv-vcf", "tumor", "somatic"],
+    frozenset(TAGS["tumor_umi_consensusfiltered.merged.cram"]): {  # UMI cram (tumor)
+        "tags": ["umi-cram", "tumor"],
         "is_mandatory": False,
         "used_by": ["deliver"],
```

Comment:
```suggestion
        "used_by": ["deliver"],
```
Is the tag `scout` refers to automatic uploads? Sometimes, for UMI customer requests, these cram files will be uploaded to scout instead.

Reply:
yes.. below answer relates to this as well.

Reply:
Then I will add the `Scout` tag to this files so we already have it for the future automatic uploads.